models/demos/t3000/llama2_70b/tt/llama_model_optimized.py

```diff
@@ -178,7 +178,7 @@ def validate_input_shape(self, inp_ids, mode):
 
         if mode == "prefill":
             assert (
-                seq_len < self.model_config["MAX_PREFILL_SEQ_LEN"]
+                seq_len <= self.model_config["MAX_PREFILL_SEQ_LEN"]
             ), f"Prefill only supports seq_len < {self.model_config['MAX_PREFILL_SEQ_LEN']}"
```

Comment:
can you change the assert string too? otherwise looks good, approved

Reply:
Ah yes thanks, fixed